Batch File/Batch File.sublime-syntax

```diff
@@ -391,7 +391,9 @@ contexts:
         - match: \n
           scope: comment.line.rem.dosbatch
           pop: true
-        - match: '[(|)]'
+          # '[(|)]' should create a illegal comment statements, but don't
+          # https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-xp/bb490986(v=technet.10)?redirectedfrom=MSDN
+        - match: ''
           scope: invalid.illegal.unexpected-character.dosbatch
```

Comment:
This is a no-op now and can just be removed.

Reply:
Thank you, this is my first pull request.

Reply:
I did remove the `match` and `scope`statement but now the syntax check fails. Should I remove the whole comment pattern? (from [383](https://github.com/sublimehq/Packages/blob/6674253653c99de02cfac68921c9a1fb3d0780d5/Batch%20File/Batch%20File.sublime-syntax#L383))

Reply:
The syntax tests need to be updated since they currently expect that the ( and ) are scoped as illegal.

BTW. I'm just pointing out the obvious issues with these changes. I don't have an opinion on whether the change itself is sane and wanted. As I understand from the discussion, the ( and ) can be illegal in some situation but not the other so I guess the choice is to either never scope them as illegal as done here, or add a lot of handling that scopes it as illegal only when it's actually illegal.

Reply:
Handling parentheses is a separate issue as it effects nearly any statement in Batch File. Scoping parens illegal in comments perse is wrong.

_Already addressed it in a local rewrite of Batch File, but got stuck at exactly this parentheses thing. It behaves so strange, that we'd probably need to write batch file twice - one version for top-level and one for parentheses. Haven't found a sane solution to do so without duplicating all the contexts. At the time work started, ST didn't support inheritance nor branching._

Reply:
I think we can all agree that Batch is a terrible language. 😅

Reply:
Same category as Perl and Bash.